api/node/routes.go

```diff
@@ -51,9 +55,21 @@ func Routes(router *gin.RouterGroup) {
 	router.GET("/statistics", Statistics)
 	router.GET("/status", StatusMetrics)
 	router.GET("/p2pstatus", P2pStatusMetrics)
+	router.GET("/godump", GoRoutinesDump)
```

Comment:
Perhaps rename to `/goroutines-dump`?

Reply:
to much to type and memorize. I would like to keep it as simple as possible